VictoriaAlbert/VictoriaAlbert/VAMTableViewController.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  VAMTableViewController.swift
+//  VictoriaAlbert
+//
+//  Created by Karen Fuentes on 11/10/16.
+//  Copyright © 2016 Karen Fuentes. All rights reserved.
+//
+
+import UIKit
+
+class VAMTableViewController: UITableViewController {
+
+    var records = [Record]()
+    var rec: Record?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        loadRecords()
+    }
+    
```

Comment:
There should never be more than one blank line, two at the most. Whatever you do, be consistent.